package.json

```diff
@@ -9,7 +9,7 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "npm run lint && tsc",
+    "build": "npm run lint && tsc && copyfiles -u 1 \"src/**/*.d.ts\" lib",
```

Comment:
We have two Fusebit `.d.ts` files under `/src`.  Since they were not included in `/lib` they weren't published.  And this leads to errors of this form:
```
node_modules/@hyperproof/hypersync-sdk/lib/common/HyperproofApiClient.d.ts:2:33 - error TS7016: Could not find a declaration file for module '@fusebit/add-on-sdk'. '/home/andrew/hp/sdk-samples/hypersync-mysql/node_modules/@fusebit/add-on-sdk/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/fusebit__add-on-sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@fusebit/add-on-sdk';`
```

See: https://stackoverflow.com/questions/56018167/typescript-does-not-copy-d-ts-files-to-build